experiments/ClimaEarth/leaderboard/leaderboard.jl

```diff
@@ -0,0 +1,168 @@
+
+        # Observational data
+        obs_var = obs_var_dict[short_name](sim_var.attributes["start_date"])
+
+        # Remove first spin_up_months from simulation
+        spin_up_months = 6
+        spinup_cutoff = spin_up_months * 31 * 86400.0
+        ClimaAnalysis.times(sim_var)[end] >= spinup_cutoff &&
+            (sim_var = ClimaAnalysis.window(sim_var, "time", left = spinup_cutoff))
+
+        obs_var = ClimaAnalysis.resampled_as(obs_var, sim_var)
+        obs_var_seasons = ClimaAnalysis.split_by_season(obs_var)
+        sim_var_seasons = ClimaAnalysis.split_by_season(sim_var)
+
+        # Add annual to start of seasons
+        obs_var_seasons = (obs_var, obs_var_seasons...)
+        sim_var_seasons = (sim_var, sim_var_seasons...)
+
+        # Take time average
+        obs_var_seasons = (
+            !ClimaAnalysis.isempty(obs_var) ? ClimaAnalysis.average_time(obs_var) : obs_var for
+            obs_var in obs_var_seasons
+        )
+        sim_var_seasons = (
+            !ClimaAnalysis.isempty(sim_var) ? ClimaAnalysis.average_time(sim_var) : sim_var for
+            sim_var in sim_var_seasons
+        )
+
+        # Save observation and simulation data
+        sim_obs_comparsion_dict[short_name] = Dict(
+            season => (sim_var_s, obs_var_s) for
+            (season, sim_var_s, obs_var_s) in zip(seasons, sim_var_seasons, obs_var_seasons)
+        )
+    end
+
+    # Filter seasons to remove seasons with no dates
+    filter!(season -> !ClimaAnalysis.isempty(sim_obs_comparsion_dict["pr"][season][1]), seasons)
+
+    # Plot bias plots
+    for compare_vars_biases in compare_vars_biases_groups
+        for season in seasons
+            fig_bias = CairoMakie.Figure(; size = (600, 300 * length(compare_vars_biases)))
+            for (row, short_name) in enumerate(compare_vars_biases)
+                sim_var = sim_obs_comparsion_dict[short_name][season][1]
+                if !ClimaAnalysis.isempty(sim_var)
+                    # Add "mean " for plotting the title
+                    sim_var.attributes["short_name"] = "mean $(ClimaAnalysis.short_name(sim_var))"
+                    ClimaAnalysis.Visualize.plot_bias_on_globe!(
+                        fig_bias,
+                        sim_obs_comparsion_dict[short_name][season]...,
+                        cmap_extrema = compare_vars_biases_plot_extrema[short_name],
+                        p_loc = (row, 1),
+                    )
+                end
+            end
+            # Do if and else statement for naming files appropriately
+            if season != "ANN"
+                CairoMakie.save(
+                    joinpath(leaderboard_base_path, "bias_$(first(compare_vars_biases))_$season.png"),
+                    fig_bias,
+                )
+            else
+                CairoMakie.save(
+                    joinpath(leaderboard_base_path, "bias_$(first(compare_vars_biases))_total.png"),
+                    fig_bias,
+                )
+            end
```

Comment:
```suggestion
                CairoMakie.save(
                    joinpath(leaderboard_base_path, "bias_$(first(compare_vars_biases))_$season.png"),
                    fig_bias,
                )
```
It's fine to change the name to _ANN